torch/csrc/jit/jit_log.cpp

```diff
@@ -32,8 +33,10 @@ class JitLoggingConfig {
   std::ostream* out;
 
   JitLoggingConfig() : out(&std::cerr) {
-    const char* jit_log_level = std::getenv("PYTORCH_JIT_LOG_LEVEL");
-    logging_levels.assign(jit_log_level == nullptr ? "" : jit_log_level);
+    const auto jit_log_level = c10::utils::get_env("PYTORCH_JIT_LOG_LEVEL");
+    if (jit_log_level.has_value()) {
+      logging_levels = jit_log_level.value();
```

Comment:
What is the default value of logging_levels? You no longer assign empty string now

Reply:
Since this is in the constructor, the default value is also ""